Issue 666: Improve User Guide for TUI Commands and Python Equivalents (#668)
pyansys/pyfluent: doc/source/user_guide/solution.rst

Selecting solution methods 
~~~~~~~~~~~~~~~~~~~~~~~~~~
The following example shows a comparison between the TUI command and the

> python code for selecting the pressure velocity coupling scheme and setting

Reviewer: ```suggestion
Python code for selecting the pressure velocity coupling scheme and setting
```

> the gradient options. Five schemes (Index-Model) are available:

Reviewer 2: ```suggestion
the gradient options. Five solution methoods (Index-Model) are available:
```

20-SIMPLE, 21-SIMPLEC, 22-PISO, 24-Coupled, and 25-Fractional Step.

> TUI command

Reviewer: ```suggestion
**TUI command**
```

.. code:: scheme

    /solve/set/p-v-coupling 24
    /solve/set/gradient-scheme yes
    /solve/set/gradient-scheme no yes 

> Python command

Reviewer: ```suggestion
**Python code**
```

    
Selecting solution controls 
~~~~~~~~~~~~~~~~~~~~~~~~~~~
The following example shows a comparison between the TUI command and the

> python code for selecting the pressure velocity controls.

Reviewer: ```suggestion
Python code for selecting the pressure velocity controls.
```

> python code for selecting the pressure velocity controls.

Reviewer: ```suggestion
Python code for selecting the pressure velocity controls.
```

TUI command

.. code:: scheme

>     /solve/set/p-v-controls 0.3 0.4

Reviewer 2: ```suggestion
    ! Set momentum and pressure under-relaxation factors
    /solve/set/p-v-controls 0.3 0.4 
```
An exclamation point is a comment char right?

Author: @dnwillia-work Semicolon char.

> Python command

Reviewer: ```suggestion
**Python code**
```

.. code:: python

    session.solver.tui.solve.set.p_v_controls(0.3,0.4) # Momentum and Pressure

Creating report definitions
~~~~~~~~~~~~~~~~~~~~~~~~~~~
The following example shows a comparison between the TUI command and the

> python code for report definitions.

Reviewer: ```suggestion
Python code for report definitions.
```

Reviewer: Does there need to be a verb? (... Python code for generating report definitions.)

> TUI command

Reviewer: ```suggestion
**TUI command**
```

.. code:: scheme

    /solve/report-definitions outlet-temp-avg surface-massavg field temperature surface-names outlet () quit

> Python command

Reviewer: ```suggestion
**Python code**
```

Initializing and solving 
~~~~~~~~~~~~~~~~~~~~~~~~
The following example shows a comparison between the TUI command and the

> python code for initialization and performing a specified number of iterations.

Reviewer: ```suggestion
Python code for initializing and performing a specified number of iterations.
```

> TUI command

Reviewer: ```suggestion
**TUI command**
```

.. code:: scheme

    /solve/initialize/hyb-initialization
    /solve/iterate 100

> Python command

Reviewer: ```suggestion
**Python code**
```
